charts/rollout-operator/templates/webhook-clusterrole-binding.yaml

```
@@ -0,0 +1,14 @@
+{{- if .Values.webhooks.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "rollout-operator.fullname" . }}-webhook-clusterrolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "rollout-operator.fullname" . }}-webhook-clusterrole
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "rollout-operator.serviceAccountName" . }}
+    namespace: {{ .Release.Namespace | quote }}
+  {{- end -}}
```

[nit] I'd align this to the starting `{{- if ...`

```suggestion
{{- end -}}
```